packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -336,6 +338,7 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
                             isCaptionsSupported={props.isCaptionsSupported}
                             /* @conditional-compile-remove(close-captions) */
                             onCaptionsSettingsClick={openCaptionsSettingsModal}
+                            onSetUserSetOverflowGalleryPosition={props.onSetUserSetOverflowGalleryPosition}
```

Comment:
I guess we are not allowing top gallery position for mobile view?

Reply:
yes is only supported for desktop, the only views that will be for mobile are "gallery" (default) "large count gallery" and "dynamic" floating local video